airframe-config/src/test/scala/wvlet/airframe/config/ConfigTest.scala

```diff
@@ -265,7 +265,7 @@ class ConfigTest extends AirframeSpec {
           unused = Some(p)
         })
 
-      unused shouldBe 'defined
+      unused shouldBe Symbol("defined")
```

Comment:
Change this to ScalaTest's `shouldBe defined` http://www.scalatest.org/user_guide/using_matchers#workingWithOptions